diff_cover/violationsreporters/violations_reporter.py

```diff
@@ -129,7 +129,7 @@ def _get_src_path_line_nodes_clover(self, xml_document, src_path):
     def _measured_source_path_matches(self, package_name, file_name, src_path):
         # find src_path in any of the source roots
         for root in self._src_roots:
-            if GitPathTool.relative_path(os.path.join(root, package_name, file_name)) == src_path:
+            if os.path.normcase(GitPathTool.relative_path(os.path.join(root, package_name, file_name))) == os.path.normcase(src_path):
```

Comment:
Doh! Thank you for this!

Reply:
Happy to help! Any idea when a release will be available with the fix?